openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2312,6 +2322,10 @@ The technology described in this specification was made available from contribut
 
    [[ To be removed from the final specification ]]
 
+   -15
+
+   * credential response always returns an array when not returning a transaction_i with the option for additional meta-data
```

Comment:
- There is a typo `transaction_i` instead of `transaction_id`
- I would add the following: 
  - `notification_id` moved to the `credentials` element 
  - Deferred Response returns `credentials` array, as well

Reply:
Agreed, should be fixed with my last commit

Reply:
Thank you Christian

(That's the first section to check whenever I read the spec :smile: )